docs/source/kinds_of_types.rst

```diff
@@ -500,7 +500,37 @@ is a *forward reference*:
    class A:
        ...
 
-Of course, instead of using a string literal type, you could move the
+Starting from Python 3.7 (:pep:`563`), you can add the special import ``from __future__ import annotations``,
+which makes the use of string literals unnecessary:
```

Comment:
```suggestion
which makes the use of string literals in annotations unnecessary:
```
Just so it contradicts the note a little less

Reply:
Yes, this looks better, thanks